serverless.yml

```diff
@@ -0,0 +1,120 @@
+service: JPC-Messaging-Service
+
+plugins:
+  - serverless-python-requirements
+
+custom:
+  serviceName: JPC-Messaging-Service
+  bucketName: 'cunningham-message-bucket'
+  shared_message_bus: ${ssm:/${self:provider.stage}/serverless/tbg_message_bus}
+  pythonRequirements:
+    pythonBin: python3
+    dockerizePip: true
+
+package:
+  patterns:
+    - "!.venv/"
+    - "!node_modules/"
+
+provider:
+  name: aws
+  runtime: python3.8
+  stage: ${opt:stage, 'dev'}
+  iamRoleStatements:
+    - Effect: Allow
+      Action:
+        - s3:*
+      Resource:
+        - 'arn:aws:s3:::${self:custom.bucketName}'
+        - 'arn:aws:s3:::${self:custom.bucketName}/*'
+    - Effect: Allow
+      Action:
+        - sqs:SendMessage
+        - sqs:SendMessageBatch
+        - sqs:ReceiveMessage
+        - sqs:ReceiveMessageBatch
+        - sqs:DeleteMessage
+        - sqs:DeleteMessageBatch
+        - sqs:GetQueueUrl
+      Resource:
+        - arn:aws:sqs:*:*:${self:custom.CunninghamQueue}
+
+  environment:
+    S3_BUCKET: ${self:custom.bucketName}
+    CUNNINGHAM_QUEUE_URL: { Ref: CunninghamQueue }
```

Comment:
What do you plan to use this queue url and bucket name for?

Reply:
Those will be gone.

Reply:
Am I logging the event correctly in my app.py file?

Reply:
Yeah, app.py looks good.